lib/fastlane/plugin/sentry/helper/sentry_helper.rb

```diff
@@ -56,6 +58,28 @@ def self.call_sentry_cli(params, sub_command)
           out_reader.value
         end
       end
+
+      def self.bundled_sentry_cli_path
+        if OS.mac?
+          self.bin_folder('sentry-cli-Darwin-universal')
+        elsif OS.windows?
+          if OS.bits == 64
+            self.bin_folder('sentry-cli-Windows-x86_64.exe')
+          else
+            self.bin_folder('sentry-cli-Windows-i686.exe')
+          end
+        else
+          if OS.bits == 64
+            self.bin_folder('sentry-cli-Linux-x86_64')
+          else
+            self.bin_folder('sentry-cli-Linux-i686')
+          end
+        end
+      end
+
+      def self.bin_folder(filename)
+        File.expand_path("../../../../../bin/#{filename}", File.dirname(__FILE__))
```

Comment:
`m`: That looks like it could break easily. Can't you somehow define the path from the project root or something?

Reply:
Not aware of a better way, as i we don't have an API to get the project root. This could break if we move the files around, but we still have the newly added unit tests that would also fail, so we have a fallback for this case.

Reply:
Thanks for explaining, @denrase. You could add a comment for that, so I won't blame you again for that in the future 😁